### PR TITLE
style: fix some typos

### PR DESCRIPTION
**Pull request by mikavilpas**
I found these using https://github.com/crate-ci/typos while I was looking at some implementation details. I think it's a quite safe change as long as you think it's worth including.

### CODE REVIEW

**Comment by MagicDuck** [comment]
Neat! Thanks for those! 😄 
Just need to update some test snapshots. You can do that with:
make prepare
make test update_screenshots=true

see https://github.com/MagicDuck/grug-far.nvim/blob/main/CONTRIBUTING.md

File: scripts/minimal_init.lua [hunk 1]
```diff
@@ -1,6 +1,8 @@
 -- Add current directory to 'runtimepath' to be able to use 'lua' files
 vim.cmd([[let &rtp.=','.getcwd()]])
 
+vim.cmd('language en_US')
```
Comment (MagicDuck) [inline scripts/minimal_init.lua:4]: Try this instead:
language en_US.UTF-8

**Comment by MagicDuck** [comment]
Are you on nvim 0.10.1? Usually the attr cell difference is caused by different nvim version.

File: scripts/minimal_init.lua (hunk 1 above)
Comment (mikavilpas) [reply scripts/minimal_init.lua]: Hmm looks like things work on my end now, but CI disagrees 

image

Comment (mikavilpas) [reply scripts/minimal_init.lua]: I'll see if there's a better way to configure my system...

**Comment by mikavilpas** [comment]
I'm on `NVIM v0.11.0-dev-1397+gfd05c7f19d` (built from source). I'll try to rebuild the snapshots with that version of neovim.

I set my environment variable to en_US now, let's see if I can fix this.

File: scripts/minimal_init.lua (hunk 1 above)
Comment (mikavilpas) [reply scripts/minimal_init.lua]: Ok, should be fixed now !

**Comment by MagicDuck** [comment]
nice! Thanks again for your contribution! 😄

**Review by MagicDuck — APPROVED** [review]

**Pull request merged**